PersonApi/V1/Factories/ResponseFactory.cs

```diff
@@ -2,12 +2,39 @@
 using PersonApi.V1.Boundary.Response;
 using PersonApi.V1.Domain;
 using System;
+using PersonApi.V1.Infrastructure.JWT;
 
 namespace PersonApi.V1.Factories
 {
-    public interface IResponseFactory
+
+
+    public interface ISnsFactory
+    {
+        PersonSns Create(Person person, Token token);
+    }
+
+    public class PersonSnsFactory : ISnsFactory
     {
-        PersonResponseObject ToResponse(Person domain);
+        public PersonSns Create(Person person, Token token)
+        {
+            return new PersonSns
+            {
+                CorrelationId = Guid.NewGuid(),
+                DateTime = DateTime.UtcNow,
+                EntityId = person.Id,
+                Id = Guid.NewGuid(),
+                EventType = "PersonCreatedEvent",
+                Version = "v1",
+                SourceDomain = "Person",
+                SourceSystem = "PersonAPI",
+                User = new User
+                {
+                    Id = Guid.NewGuid(),
```

Comment:
Is this property actually needed if we set an arbitrary value in it?

Reply:
User information is coming from the JWT token. It doesn't include an ID (for now), so I am using an arbitrary value until it includes one in the future (I hope)

Reply:
I'd have just left it empty, but I guess that's just as arbitrary.